composer.json

```diff
@@ -23,9 +23,10 @@
         "keboola/billing-api-php-client": "^0.1",
         "keboola/configuration-variables-resolver": "^1.0.0",
         "keboola/dockerbundle": "dev-odin-library-3@dev",
-        "keboola/input-mapping": "^13.2",
+        "keboola/input-mapping": "^14.0",
         "keboola/job-queue-internal-api-php-client": "^9.4",
         "keboola/object-encryptor": "^0.3.2",
+        "keboola/output-mapping": "dev-erik-PS-3117 as 16.0.1",
```

Comment:
pak to vymenim

Reply:
to už můžeš vyměnit hned, to nepotřebuje čekat až dojede master

Reply:
done